apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3043,7 +3045,7 @@ static void tc_libc_stdio_dprintf_invalid_fd_n(void)
 
 	/* Testcase */
 	ret = dprintf(CONFIG_NFILE_DESCRIPTORS, "%s", str);
-	TC_ASSERT_EQ("dprintf", ret, 0);
+	TC_ASSERT_NEQ("dprintf", errno, 0);
```

Comment:
I think you'd better check with `EBADF`.

Reply:
Okay. modify